iceoryx_posh/test/integrationtests/test_posh_mepoo.cpp

```diff
@@ -374,64 +376,48 @@ TEST_F(Mepoo_IntegrationTest, MempoolConfigCheck)
-TEST_F(Mepoo_IntegrationTest, DISABLED_MempoolOver4GB)
-{
-    constexpr uint32_t chunkSize = 16777216;
-    constexpr uint32_t m_numChunks = 350;
-
-    MemPoolInfoContainer memPoolTestContainer;
-
-    std::vector<TestMemPoolConfig> testMempoolConfig;
-    testMempoolConfig.push_back({chunkSize, m_numChunks});
-    auto mempoolSize = 1234; // TODO get from createRouDiConfig();
-    SetUp(memPoolTestContainer, testMempoolConfig, configType::CUSTOM);
-
-    uint64_t size = 666; // this->m_roudiEnv->m_roudiApp->m_shmMgr.m_mePooConfig.getSharedMemorySize();
-    EXPECT_THAT(size, Eq(mempoolSize));
-
-    constexpr int samplesize2 = chunkSize - 10;
-    const int repetition2 = 2;
-    ASSERT_TRUE(sendreceivesample<samplesize2>(repetition2));
-
-    auto mempolIndex2 = indexOfMempool<samplesize2>(testMempoolConfig);
-    memPoolTestContainer[mempolIndex2].m_usedChunks = repetition2;
-    memPoolTestContainer[mempolIndex2].m_minFreeChunks = m_numChunks - repetition2;
-
-    m_roudiEnv->InterOpWait();
-
-
-    // get mempoolconfig from introspection
-    MemPoolInfoContainer memPoolInfoContainer;
-    EXPECT_THAT(compareMemPoolInfo(memPoolInfoContainer, memPoolTestContainer, Log::Off), Eq(false));
-    getMempoolInfoFromIntrospection(memPoolInfoContainer);
-
-    EXPECT_THAT(compareMemPoolInfo(memPoolInfoContainer, memPoolTestContainer), Eq(true));
-}
-
-TEST_F(Mepoo_IntegrationTest, DISABLED_WrongSampleSize)
+TEST_F(Mepoo_IntegrationTest, WrongSampleSize)
 {
     MemPoolInfoContainer memPoolTestContainer;
-
     auto testMempoolConfig = defaultMemPoolConfig();
     SetUp(memPoolTestContainer, testMempoolConfig, configType::CUSTOM);
-
     constexpr int samplesize3 = 2048;
     const int repetition3 = 1;
-
-    EXPECT_DEATH({ sendreceivesample<samplesize3>(repetition3); }, ".*");
+    auto errorHandlerCalled{false};
+    iox::Error receivedError;
+    auto errorHandlerGuard = iox::ErrorHandler::SetTemporaryErrorHandler(
+        [&errorHandlerCalled,
+         &receivedError](const iox::Error error, const std::function<void()>, const iox::ErrorLevel) {
+            errorHandlerCalled = true;
+            receivedError = error;
+        });
+
+    sendreceivesample<samplesize3>(repetition3);
+
+    EXPECT_TRUE(errorHandlerCalled);
+    ASSERT_THAT(receivedError, Eq(iox::Error::kMEPOO__MEMPOOL_GETCHUNK_CHUNK_IS_TOO_LARGE));
 }
 
-TEST_F(Mepoo_IntegrationTest, DISABLED_SampleOverflow)
+TEST_F(Mepoo_IntegrationTest, SampleOverflow)
 {
     MemPoolInfoContainer memPoolTestContainer;
-
     auto testMempoolConfig = defaultMemPoolConfig();
     SetUp(memPoolTestContainer, testMempoolConfig, configType::CUSTOM);
-
     constexpr int samplesize1 = 200;
-    const int repetition1 = DefaultNumChunks;
-    EXPECT_DEATH({ sendreceivesample<samplesize1>(repetition1); }, ".*");
+    const int repetition1 = 2 * DefaultNumChunks;
+    auto errorHandlerCalled{false};
+    iox::Error receivedError;
```

Comment:
same here, please initialize to `NO_ERROR`

Reply:
Done.